src/binding.cc

```diff
@@ -15,7 +15,8 @@
  */
 
 #include "iconv.h"
-#include "nan.h"
+#include "napi.h"
+#include "uv.h"
 #include "node_buffer.h"
```

Comment:
@MrKashlakov I believe you can remove this include, because node-addon-api proivdes access to `node::Buffer` via N-API.